app/src/main/java/org/openobservatory/ooniprobe/model/database/TestDescriptor.kt

```diff
@@ -172,3 +184,61 @@ class NettestConverter : TypeConverter<String, Any>() {
     ).toList()
 }
 
+class ITestDescriptor(
```

Comment:
What does the prefix `IT` mean in this context?

Reply:
The `I` is to denote its a type for interaction only. Not functional. It provides the proper types for all `TestDescriptor` properties.

Reply:
gotcha!